storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operations/execute/PreparedExecuteSQL.java

```diff
@@ -113,7 +118,22 @@ public Object executeAsBlocking() {
     @CheckResult
     @Override
     public Observable<Object> asRxObservable() {
-        return RxJavaUtils.createObservable(storIOSQLite, this);
+        throwExceptionIfRxJavaIsNotAvailable("asRxObservable()");
+
+        final Set<String> observesTables = rawQuery.observesTables();
+        final Set<?> observesTags = rawQuery.observesTags();
+
+        final Observable<Object> observable;
+        if (!observesTables.isEmpty() || !observesTags.isEmpty()) {
+            observable = ChangesFilter.applyForTablesAndTags(storIOSQLite.observeChanges(), observesTables, observesTags)
```

Comment:
Yep, I agree that it is the side effect operation and it will not be clean to subscribe on some changes with it. 
But current `RawQuery.observesTags` possibility can be misleading.
Maybe we should note this feature in doc, because it may look as unexpected behavior =(